packages/ping-protect/package.json

```diff
@@ -20,6 +20,6 @@
   },
   "types": "./dist/index.ts.d.ts",
   "dependencies": {
-    "@forgerock/javascript-sdk": "workspace:*"
+    "@forgerock/javascript-sdk": "workspace:^"
```

Comment:
Should there be a version number after this? What does the caret mean when it's alone like this?